rakefile

```diff
@@ -612,4 +612,44 @@ namespace :pe_xl do
   task :deploy => [:clean, :provision, :run_plan] do
     puts "Deploying a pe_xl architecture"
   end
+
+end
+
+namespace :validate do
+  desc "Validate the environment variables"
+  task :validate_performance_env_vars do
+    Rake::Task["bigquery:verify_baseline_version"].invoke
+  end
+
+  desc "Verifies that the BEAKER_INSTALL_TYPE environment variable and dependencies have been specified"
+  task :verify_beaker_install_type do
```

Comment:
From a ux perspective, I would prefer that we standardize on "validate" or "verify" and use that prefix as the namespace.  The prefix should then not be used in the namespaced task name.  This would enable me to execute the tasks relative to the namespace.

E.G.
```
bundle exec rake validate:performance_env_vars
bundle exec rake validate:beaker_install_type
bundle exec rake validate:baseline_version
```

Reply:
I was trying to differentiate between validation vs. verification but I agree that it is confusing from a UX perspective. I've renamed based on your suggestion; thanks.